internal/tools/tools.go

```diff
@@ -15,17 +15,24 @@ import (
 	_ "github.com/frapposelli/wwhrd"
```

Comment:
Do we really need all the new tools? I only see references to `protoc` on the tasks, can you clarify?